### ISSUE TITLE
Not all host IPs are added to ipcache

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#8167]
Example:

ipcache:
```
10.16.0.1/32             1 0 0.0.0.0
f00d::a10:0:0:1/128      1 0 0.0.0.0
fd00::b/128              1 0 0.0.0.0
192.168.34.11/32         1 0 0.0.0.0
```

addresses:
```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
2: enp0s3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    inet 10.0.2.15/24 brd 10.0.2.255 scope global dynamic enp0s3
       valid_lft 85884sec preferred_lft 85884sec
3: enp0s8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    inet 192.168.33.11/24 brd 192.168.33.255 scope global enp0s8
       valid_lft forever preferred_lft forever
4: enp0s9: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    inet 192.168.34.11/24 brd 192.168.34.255 scope global enp0s9
       valid_lft forever preferred_lft forever
5: docker0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default
    inet 172.17.0.1/16 brd 172.17.255.255 scope global docker0
       valid_lft forever preferred_lft forever
```

**Comment by joestringer** [comment]
@tgraf This is resolved in v1.6 already right?

**Comment by tgraf** [comment]
Yes! Fixed by @#8243
